docs/demo.rst

```diff
@@ -59,8 +59,8 @@ Now we can download some example data, which we'll use for our demo:
 
 .. code-block:: bash
 
-    (ga4gh-env) $ wget http://www.well.ox.ac.uk/~jk/ga4gh-example-data-v2.1.tar
-    (ga4gh-env) $ tar -xvf ga4gh-example-data-v2.1.tar
+    (ga4gh-env) $ wget http://www.well.ox.ac.uk/~jk/ga4gh-example-data-v3.0.tar
```

Comment:
sphinx probably has some notion of variables, right, so we don't need to change this in two places going forward?